src/gui/optionsdialog.cpp

```diff
@@ -280,7 +280,7 @@ void OptionsDialog::loadBehaviorTabOptions()
     m_ui->checkShowSplash->setChecked(!pref->isSplashScreenDisabled());
     m_ui->checkProgramExitConfirm->setChecked(pref->confirmOnExit());
     m_ui->checkProgramAutoExitConfirm->setChecked(!pref->dontConfirmAutoExit());
-    m_ui->checkConfirmPauseAndResumeAll->setChecked(pref->confirmPauseAndResumeAll());
+    m_ui->checkConfirmStopAndResumeAll->setChecked(pref->confirmPauseAndResumeAll());
```

Comment:
`Resume` should be renamed side-by-side.

Reply:
There are also related states (paused/resumed) that should be renamed too.

Reply:
>`Resume` should be renamed side-by-side.

What?

>There are also related states (paused/resumed) that should be renamed too.

Could you point me to one of them?

Reply:
> > `Resume` should be renamed side-by-side.
> 
> What?

I mean that "resume" isn't good "opposite action" for "stop" so it should be renamed to something more appropriate (e.g. "start") at the same time with renaming "pause" to "stop".

Reply:
>I mean that "resume" isn't good "opposite action" for "stop

I don't see it but i don't mind either way. I'll wait a day to see if there's any opposition to renaming `resume` -> `start`.

Reply:
> > There are also related states (paused/resumed) that should be renamed too.
> 
> Could you point me to one of them?

Here is what lies in plain sight:
![000](https://github.com/qbittorrent/qBittorrent/assets/5063477/2209b881-5771-43bd-91d9-ec0779e2fbea)

And there are other possible occurrences in UI and code.

Reply:
>so it should be renamed to something more appropriate (e.g. "start")

Done.

Reply:
> > BTW, even if it is decided to leave "Resume" as the action name, "Resumed" has always annoyed me as the name of the state, so I would still prefer to see something like "Working" or "Running" instead of it.
> 
> I don't mind which one as long as it is decided before i start. I can't just do a search and replace for this term...

I would prefer "Running" because many people associate the term "Working" with an indicator of torrent "healthy", so it can confuse them.

Reply:
>I would prefer "Running"

As a state yes.. but as a verb? "Run all" "Run torrents". Doesn't sound good to me.

Reply:
> > I would prefer "Running"
> 
> As a state yes.. but as a verb? "Run all" "Run torrents". Doesn't sound good to me.

Only as a state.
It seems OK for me to use "start" and "running" together (we "start" some torrent and then it's "running"). "Started" seems not so good for me as continuous state.
Maybe we can even use "Run" instead of "Start".

Reply:
> Maybe we can even use "Run" instead of "Start".

"Start" is best left.

Reply:
> It seems OK for me to use "start" and "running" together (we "start" some torrent and then it's "running"). "Started" seems not so good for me as continuous state.

👍